src/Passes/Source/Rules/FactoryConfig.hpp

```diff
@@ -76,12 +76,17 @@ namespace quantum
 
         bool optimiseBranchQuantumOne() const
         {
-            return optimise_branch_quatum_one_;
+            return optimise_branch_quantum_one_;
         }
 
         bool optimiseBranchQuantumZero() const
         {
-            return optimise_branch_quatum_zero_;
+            return optimise_branch_quantum_zero_;
+        }
+
+        bool optimiseSelectQuantumOne() const
+        {
+            return optimise_select_quantum_zero_;
         }
```

Comment:
Suspicious function:
The function name ends with `One` but the returned value name ends with `zero`. Did you mean what is below?
```suggestion
        bool optimiseSelectQuantumOne() const
        {
            return optimise_select_quantum_one_;
        }
```